src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -29,7 +29,6 @@
         background-color: white;
         border: 1px solid #cce3fe;
         border-radius: 5px;
-        color: inherit;
```

Comment:
Did some quick testing and it looks okay to remove this. Visited color works correctly (fixes the initial ticket) and the color still looks fine when the code snippet is not a link.